src/node_report.cc

```diff
@@ -602,17 +602,22 @@ void PrintNativeStack(FILE* fp) {
         IMAGEHLP_LINE64 line;
         line.SizeOfStruct = sizeof(line);
         if (SymGetLineFromAddr64(hProcess, dwAddress, &dwOffset, &line)) {
-          fprintf(fp, "%2d: [pc=0x%p] %s [+%d] in %s: line: %lu\n", i, pSymbol->Address, pSymbol->Name, dwOffset, line.FileName, line.LineNumber);
+          fprintf(fp, "%2d: [pc=0x%p] %s [+%d] in %s: line: %lu\n", i, 
+			reinterpret_cast<void*>(pSymbol->Address), pSymbol->Name, 
+			dwOffset, line.FileName, line.LineNumber);
         } else {
           // SymGetLineFromAddr64() failed, just print the address and symbol
           if (dwOffset64 <= 32) { // sanity check
```

Comment:
nb. Consistently dropping into the else clause here is why only `test-api.js` is failing in nodejs/nodereport#26 as it's succeeding the `SymGetLineFromAddr64` call -- The other tests fail the call and then fail the sanity check. The original `fprintf` for the else clause of the sanity check doesn't have parameters after the `pSymbol->Address` argument, so there the `fprintf` call there didn't crash.